### PR TITLE
Spacy pipeline component.

### PR DESCRIPTION
**Pull request by Govind9**

### CODE REVIEW

**Comment by lintool** [comment]
@nikhilro can you take a look? thanks!

File: scripts/util.py [hunk 1]
```diff
@@ -175,16 +136,14 @@ def add_vocab_csvs(data_dir: Path, nlp, append: bool = True):
...
+        wb_type = file.split('wikidata_')[1].split('s.csv')[0].upper()
+        water_bodies[wb_type] = []
+        df = pd.read_csv(file)
+        for i in range(len(df)):
+            if (type(df['Name'][i]) is str and
+            len(df['Name'][i]) > 1 and
+            (not re.search('^Q[0-9]+', df['Name'][i]))):
+                names = name_split(df['Name'][i])
+                # split 'amazon river' or 'river amazon' to 'amazon'
+                for name in names: water_bodies[wb_type].append((name, df['ID'][i]))
```
Comment (nikhilro) [inline scripts/util.py]: nit pick: no one liners please. best to keep formatting consistent

File: waterwheel/waterwheel.py [hunk 2]
```diff
@@ -1,149 +1,200 @@
...
-        self.vocab = {}
-        self.wikidata = {}
-        self.stop_words = set()
-        
-        self.load_features_data() 
+        super().__init__(nlp, phrase_matcher_attr='LOWER', overwrite_ents=overwrite_ents)
+        self._ent_ids = defaultdict(lambda: "WATER_BODY")
+        self._stop_words = set()
+        self._wikidata = {}
+        self._doc_bins = {}
```
Comment (nikhilro) [inline waterwheel/waterwheel.py]: :100: on variable names!

File: scripts/util.py (hunk 1 above)
Comment (Govind9) [reply scripts/util.py]: Made the change.

**Pull request merged**